### PR TITLE
Update MySQL Stored Proc call BBE

### PR DESCRIPTION
**Pull request by Shairam**

### CODE REVIEW

File: examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal [hunk 1]
```diff
@@ -81,6 +81,11 @@ function insertRecord(mysql:Client sqlClient) {
     if (retCall is sql:ProcedureCallResult) {
         io:println("Call stored procedure `InsertStudent` is successful : ", 
                 retCall.executionResult);
+
+        error? e = retCall.close();
```
Comment (niveathika) [inline examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal:85]:
```suggestion
        _ = retCall.close();
```
Update in other places as well

Comment (Shairam) [reply examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal]: Yes noted. I will update them. Thank you.

**Comment by niveathika** [comment]
@Shairam There is no rule that entire PR should be only one commit. Undoing the committing and force pushing would be fine for single PR. However, if it is a collaboration between multiple people, there may be conflicts. If there is anything to fix in the PR add another commit do not undo the already pushed commit

**Review by niveathika — APPROVED** [review]

**Comment by Shairam** [comment]
@niveathika  Yes sure. I will do as suggested in the future. Thank you so much.

**Pull request merged**